### PR TITLE
Home task HTML_03

### PR DESCRIPTION
**Pull request by IrynaSafonava**
Hello Siarhei,
I had difficulties during pullin and then committing the changes. Not easy to deal with the GIT. Hope this has not affected any records.
Thank you !

### CODE REVIEW

**Comment by IrynaSafonava** [comment]
Did not get exactly what i had done here ;-) [Merge branch 'm-sc1-59-19' of https://github.com/IrynaSafonava/sc.it-academy.by into m-sc1-59-19]

File: isafonova/HTML_02/task_Python_2.2.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+		<code>print len(months)</code>
+		<p>которая вернёт 12.</p>
+		<p>Ещё один пример списков — категории в поваренной книге. Например...</p>
+		<code>categories = ['Main dish','Meat','Fish','Soup','Cookies']</code>
+		<p>Таким образом, в <strong>categories[0]</strong> хранится '<i>Main dish</i>', а в <strong>categories[4]</strong> — '<i>Cookies</i>'. Всё очень просто. Я уверен, вы сможете придумать много других примеров использования списков.</p>
+		<hr>
+		<h2 id="comments">Комментарии</h2>
+		<p><a href="#summary">К содержанию</a></p>
+		<p>Далее, мы должны обсудить комментарии. Комментарии важны по нескольким причинам. Они не только говорят вам или другим людям, что вы пытаетесь делать, но и напомнят вам через полгода, что же вы хотели сделать. Когда вы будете писать много программ, комментарии станут важными. Также при их помощи вы заставите Python игнорировать какие-либо части кода. Чтобы закомментировать строку, поставьте перед ней знак '#'. Например:</p>
+		#
```
Comment (pluhin) [inline isafonova/HTML_02/task_Python_2.2.html:40]: It is so alone :) Where is pre?

**Review by pluhin — APPROVED** [review]
Very, good! Approved

File: isafonova/HTML_02/task_Python_2.2.html (hunk 1 above)
Comment (IrynaSafonava) [reply isafonova/HTML_02/task_Python_2.2.html:40]: fixed

**Pull request merged**